src/System.Security.Permissions/src/System/Security/CodeAccessPermission.cs

```diff
@@ -11,14 +11,14 @@ public abstract partial class CodeAccessPermission : IPermission, ISecurityEncod
         public void Assert() { }
         public abstract IPermission Copy();
         public void Demand() { }
-        [Obsolete("Deny is obsolete and will be removed in a future release of the .NET Framework. See http://go.microsoft.com/fwlink/?LinkID=155570 for more information.")]
+        [Obsolete]
```

Comment:
They're relevant if other code in the same assembly calls them.  But I just figured we'd want to keep obsolete attributes consistent between ref and src.  My care factor is low, though.